caasp-stack.yaml

```diff
@@ -70,6 +83,58 @@ parameters:
     description: Root Password for the VMs
     default: linux
 
+  cpi_auth_url: 
+    type: string
+    description: Cloud Provider Interface - keystone auth url
+    default: https://192.168.24.26:5000/
+          
+  cpi_domain_name: 
+    type: string
+    description: Cloud Provider Interface - keystone domain name
+    default: Default
+    #constraints:
+    #  - custom_constraint: keystone.domain
+
+  cpi_tenant_name:
+    type: string
+    description: Cloud Provider Interface - keystone project
+    default: admin
+    #constraints:
+    #  - custom_constraint: keystone.project
+
+  cpi_region:
+    type: string
+    description: Cloud Provider Interface - keystone region
+    default: region1
+    #constraints:
+    #  - custom_constraint: keystone.region
+
+  cpi_username:
+    type: string
+    description: Cloud Provider Interface - keystone username
+    default: admin
+    #constraints:
+    #  - custom_constraint: keystone.user
+
+  cpi_password:
+    type: string
+    description: Cloud Provider Interface - keystone user password
+
+  cpi_lb_mon_retries:
+    type: number
+    description: Cloud Provider Interface - monitoring retries
+    default: 3
+    
+  cpi_cinder_bs_version:
+    type: string
+    description: Cloud Provider Interface - cinder block storage version
+    default: '1'
```

Comment:
Valid values are v1, v2, v3 and auto

Reply:
thx! done added constraint, fixed parameter name